app/models/pawn.rb

```diff
@@ -1,4 +1,38 @@
 class Pawn < Piece
-	def move
+
+	#en passant status of self
+	def en_passant_status(end_x, end_y)
+		if (self.pos_y - end_y).abs == 2
+			self.update_attributes(en_passant: true)
+		else
+			self.update_attributes(en_passant: false)
+		end
+	end
+
+	#determine if pawn can capture opposing pawn en passant
+	def can_capture_en_passant(opponent_pawn)
+		if opponent_pawn.en_passant == true && opponent_pawn.updated_at == @game.updated_at
+			return true
+		end
+	end
+
+	#do the capture en passant move
+	def capture_en_passant(opponent_pawn)
```

Comment:
at some point you will actually need to verify that en_passant is a valid move so that you can inform the UI.  I'm not sure i'm seeing that but just an FYI